appconf.py

```diff
@@ -228,7 +230,9 @@ def generate_tls_config(cert):
 
 def activate_lets_encrypt(server_name):
     with open('/tmp/le-domain.txt', 'w') as f:
-        f.write(server_name)
+        f.write('-d ' + ' -d '.join(server_name.split(' ')))
+    if len(names := server_name.split(' ')) > 1:
+        server_name = names[0]
```

Comment:
Alternatively:
```
names = server_name.split(' ')
with open('/tmp/le-domain.txt', 'w') as f:
    ' '.join(f'-d {name}' for name in names)
server_name = names[0]
```
Or even just use `names[0]` below...

I have nothing against the walrus but it seems a bit forced here, given you do the same thing just a line before ;)

Reply:
![image](https://user-images.githubusercontent.com/9108586/75784027-daa9e680-5d61-11ea-8a25-2a430b9c3a75.png)